docs/source/index.rst

```diff
@@ -11,14 +11,26 @@ with Numba, we suggest you start with the :doc:`User manual <user/index>`.
 
 
 .. toctree::
-   :numbered:
+   :caption: For all users
    :maxdepth: 2
 
    user/index.rst
    reference/index.rst
+
+
+.. toctree::
+   :caption: For GPU users
```

Comment:
Perhaps this should differentiate between CUDA and ROCm GPU users. There's the cuda guide and reference that could go under cuda.

Reply:
If we tried to differentiate, then ROCm would end up with only one line below the caption. Since there are only 3 items below this caption, personally I wouldn't split them further.

Reply:
I find it strange to lump different technologies together based on target hardware kind. I personally am ok with one line for ROCm and two for CUDA and have built the docs with this in and it seems ok. Anyone else have a view?